frontend/src/features/logic/utils/isConditionFulfilled.ts

```diff
@@ -78,6 +78,11 @@ export const isConditionFulfilled = (
         }
         return condition.value === currentValue.value
       }
+      // In angular, number equality is string=== but decimal equality is number===.
+      // Need to replicate this behavior for backward-compatibility.
+      if (fieldType === BasicField.Decimal) {
```

Comment:
Not a comment on your PR, but why are `input` and `fieldType` accessed onto `args` in the uses above? 🤔

Reply:
i think `isLogicableField` is a type guard, and it does something to restrict the type of `args.input` and `args.fieldType`

Reply:
Thanks! Can we verify that actually? It looks like the other uses are themselves in checks (e.g. `if (isRadioFormFieldValue(currentValue, args.fieldType))`, so I'm not sure an access from `args` really matters? 🤔 

And I guess my question was to highlight consistency. This check here is using `fieldType` rather than `args.fieldType`, and it's not clear whether there's purpose in the differing uses.